test/e2e/test_groups/login_group/login_tests.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'Keystone should allow admins to login to the Admin UI' : function (client) {
+    client
+      .url('http://XPS8500:3000/keystone')
```

Comment:
I only wanted it to work in my machine...crap=)  Hopefully will have time to address all the comments this evening (EST).